examples/Manifest.toml

```diff
@@ -240,8 +246,10 @@ uuid = "c8b6d40d-e815-466f-95ae-c48aefa668fa"
 version = "0.7.1"
 
 [[deps.ClimaTimeSteppers]]
-deps = ["CUDA", "ClimaComms", "DataStructures", "DiffEqBase", "DiffEqCallbacks", "KernelAbstractions", "LinearAlgebra", "SciMLBase", "StaticArrays"]
-git-tree-sha1 = "deb2bc80d429ec991bb276aad8b4370d2c4be4dc"
+deps = ["CUDA", "ClimaComms", "DataStructures", "DiffEqBase", "DiffEqCallbacks", "KernelAbstractions", "Krylov", "LinearAlgebra", "LinearOperators", "SciMLBase", "StaticArrays"]
+git-tree-sha1 = "8a79c326bf92abb8ff8627fc529304d238ea1e94"
+repo-rev = "dy/newton_krylov"
```

Comment:
Is that branch something that could be merged into ClimaTimesteppers?

Reply:
Yeah I was just waiting to see if I'd need to tinker with JFNK any further to get it to work well. For very nonlinear problems (like those involved in the land model) JFNK is often appended with a line search method, but my experiments so far seem to indicate that it won't be necessary for IMEX EDMF. I'll go ahead and merge it in today.